### ISSUE TITLE
Update to PhET's Accessibility Page

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/website#336]
We would like to add a link to PhET's accessibility page (http://phet.colorado.edu/en/about/accessibility) that links to implementation notes about the recent Parallel DOM approach.  It would be great if we could get this in for the next deploy of the live site.

html document is here, though I am not sure if @emily-phet would prefer this be exported to pdf:
http://www.colorado.edu/physics/phet/dev/accessibility/accessibility-documentation.html

I am also unsure where the link should be on the page, and if it should be introduced by other content.  Assigning to @emily-phet if she has a preference on those items.

**Comment by jessegreenberg** [comment]
This is being handled along with other updates in #412, closing.
